js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```
@@ -3253,6 +3253,8 @@ organization=Organization
 send=Send
 redirectWhenEmailMatches=Redirect when email domain matches
 redirectWhenEmailMatchesHelp=Automatically redirect the user to this identity provider when the email domain matches the domain
+redirectWhenEmailMatchesAny=Redirect when email domain matches any of the organizations domains
+redirectWhenEmailMatchesAnyHelp=Automatically redirect the user to this identity provider when the email domain matches any of the organizations domains
```

Nitpick: As this help fragment is a full sentence, it should end with a full-stop.

```suggestion
redirectWhenEmailMatchesAnyHelp=Automatically redirect the user to this identity provider when the email domain matches any of the organizations domains.
```

---

applied, thanks